geometry_msgs/PointStamped[] objects
int32 quantity
